src/errors.rs

```diff
@@ -0,0 +1,36 @@
+
+use std::error::Error;
+use std::fmt;
+
+// TODO: distinguish between UnknownLicenseId and UnknownExceptionId
+// after https://github.com/nikomatsakis/lalrpop/issues/113 is resolved
+
+#[derive(Debug, Clone, Copy)]
+pub enum SpdxParseError<'a> {
```

Comment:
I think keeping this name as `ParseError` is more consistent with Rust's naming conventions, which are generally fine with conflicts with names from other modules. As you mentioned, you can use `as` to mask names on import to avoid a conflict.